app/src/main/java/org/mozilla/tv/firefox/experiments/ExperimentExceptions.kt

```diff
@@ -11,3 +11,11 @@ package org.mozilla.tv.firefox.experiments
  * considered an illegal experiment state
  */
 open class ExperimentIllegalStateException(message: String) : IllegalStateException(message)
+
+/**
+ * [NotInExperimentException] is for logging users that are currently not part of an
+ * Experiment
+ *
+ * @param name: Name of the experiment
+ */
```

Comment:
Nit: might be good to put a note here that this can happen on the first launch after an experiment has been created, since it uses persisted values.

Reply:
@Baron-Severin Hm, not sure if that is _the_ use case (I personally like to keep it as general as possible). We should be looking to optimize that in the future when we are looking at reducing launch time fwiw. What do you think?

Reply:
I haven't spent a lot of time on Fretboard, so correct me if I'm wrong here.  I didn't think it waited for a response on startup?  My understanding was that it would just use the previously persisted value, and in the background make a request and store a new one.  So we would pretty much guarantee that we were using the Fretboard data pulled during our previous session, but also that it wouldn't have any significant impact on launch time.

Agreed that we shouldn't describe that as the only thing that can cause this error.  Other problems could definitely occur as well.  But it's a known issue that isn't immediately obvious, and anyone investigating this crash will probably look in this file first.  I think it would be good to mention it as a possibility.

Reply:
@Baron-Severin You're right, we are still blocking the UI thread with internal data storage load though. I still think there can be a reactive solution to this while finding a way to synchronize experiments load with fetch.

Having said that, I think it's a feasible possibility that an user is not part of any experiments :)

Reply:
I'll add a follow-up story and a reference comment :)

Reply:
For sure.  I was thinking something like "This can happen when a user is not part of any experiment, or they have not yet fetched updated data from Fretboard."